spec.bs

```diff
@@ -3193,6 +3188,20 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=reporting context map=]
 1. Let |response| be the [=server auction response=] which is the result of
    deserializing |auctionConfig|'s [=auction config/server response=]
    with |requestContext| according to [Section 2.3.5 of the Bidding and Auction Services IETF standard](https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#name-parsing-a-response).
+1. If |response|'s [=server auction response/nonce=] [=map/exists=]:
+  1. Let |capturedAuctionNonceHeaders| be |global|'s [=associated Document's=] [=node navigable's=]

```

Comment:
is nonce always lowercase in the result nonce header?

Reply:
It may make sense to require that. The current implementation in Chrome is not opinionated about that, but that is something we can change.

Reply:
if we think we may want to change it in the impl, can we create a bug (or a github issue and a TODO in the spec) for it so that we don't forget about the mismatch between impl and spec?

Reply:
Added http://crbug.com/396634015